packages/paste-core/components/list/src/index.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import {Text, NULLIFY_TEXT_PROPS} from '@twilio-paste/text';
+
+interface BaseListProps extends React.OlHTMLAttributes<''> {
```

Comment:
This is now effectively typed as an HTML list (there is no UlHTMLAttributes, I assume they are identical). So you can apply any HTML attribute that is relevant to being on a list and we can safely spread those onto `Text`